Doc/Tutorial/chapter_uniprot.tex

```diff
@@ -25,15 +25,15 @@ \subsection{Parsing Swiss-Prot records}
 >>> from urllib.request import urlopen # Python 3 only
 >>> from io import TextIOWrapper
 >>> 
->>> handle = TextIOWrapper(urlopen("http://example.org/myprositefile.dat"))
+>>> handle = TextIOWrapper(urlopen("https://raw.githubusercontent.com/biopython/biopython/master/Tests/SwissProt/F2CXE6.txt"))
```

Comment:
Sorry, never mind, reading the text about this example ought to have been a SwissProt text file in the first place.